site2/docs/getting-started-standalone.md

```diff
@@ -106,7 +106,7 @@ pulsar-io-aerospike-{{pulsar:version}}.nar
 > **Note**  
 > * If you are running Pulsar in a bare metal cluster, make sure `connectors` tarball is unzipped in every pulsar directory of the broker (or in every pulsar directory of function-worker if you are running a separate worker cluster for Pulsar Functions).
 > 
-> * If you are [running Pulsar in Docker](getting-started-docker.md) or deploying Pulsar using a docker image (e.g. [K8S](deploy-kubernetes.md) or DC/OS, you can use the `apachepulsar/pulsar-all` image instead of the `apachepulsar/pulsar` image. `apachepulsar/pulsar-all` image has already bundled [all builtin connectors](io-overview.md#working-with-connectors).
+> * If you are [running Pulsar in Docker](getting-started-docker.md) or deploying Pulsar using a docker image (e.g. [K8S](deploy-kubernetes.md) or [DC/OS](https://dcos.io/), you can use the `apachepulsar/pulsar-all` image instead of the `apachepulsar/pulsar` image. `apachepulsar/pulsar-all` image has already bundled [all builtin connectors](io-overview.md#working-with-connectors).
```

Comment:
If we're going to deprecate DC/OS content, should we remove the `or DC/OS` here instead of adding the link?

Reply:
Some users might still use DC/OS to deploy Pulsar. What I want is not to provide instructions in our docs since the content is inaccurate and many links are broken, while users can do it in their way. Thoughts?

Reply:
I agree with that approach @Anonymitaet.